common.blocks/page/_view/page_view_500.bemtree.js

```diff
@@ -0,0 +1,19 @@
+block('page').mod('view', '500').content()(function() {
```

Comment:
Why did you wrote that tags from error-board.bemtree.js should be set in BEMHTML but here they are in BEMTREE? Or for such static page like 505 it is ok?